core/data.go

```diff
@@ -505,6 +505,43 @@ type PaymentMetadata struct {
 	CumulativePayment *big.Int `json:"cumulative_payment"`
 }
 
+func NewPaymentMetadata(
+	// account that the payment is for. must not be a 0 address
+	accountID gethcommon.Address,
+	// the time of the dispersal, which is stored as a unix nano timestamp
```

Comment:
```suggestion
	// the time of the dispersal
```

Reply:
nit: technically not a unix nano ts right? its got all the other stuff, like the monotonic timestamp part for example

Reply:
> all time instances have to have a unix ts part right? its only the monotonic part that can be missing?

Right.

> although passing ts seems fine tbh

Alright, I decided just to make the doc more explicit, but skipped the todo.

7cac5af5732826a1a138c9e565feb27369dfb62b